### ISSUE TITLE
Help people on tech support make better use of Honeycomb

### ISSUE THREAD

**Issue by lucyb** [opensafely-core/sysadmin#85]
Run a training session, explaining what information is available and showing how to build common queries?

**Comment by sebbacon** [comment]
Other options:
* Make a video about it people can rewatch
* Review dashboards we currently have; e.g. thematic dashboards; add new things to existing dashboards; prune less useful things from existing dashboards (all modulo a point Simon made about dashboards not generally being sufficient; that would be "be able to make your own charts")

**Comment by madwort** [comment]
Workshop is better than a video? Prepare a set of curated questions.

**Comment by sebbacon** [comment]
I agree, but for new starters etc it might be good to prepare something recorded too. E.g. an intro video that we start the workshop with? Just an idea.

**Comment by lucyb** [comment]
New suggestions
1. New issue - More detailed documentation of the data model that we're sending to honeycomb
2. Workshop(?) to get people familiar with using the honeycomb UI, inc list of honeycomb videos to watch
3. New issue - write a blog post about this work
